webapi/src/main/scala/org/knora/webapi/messages/admin/responder/listsmessages/ListsMessagesADM.scala

```diff
@@ -480,12 +469,11 @@ case class NodeADM(nodeinfo: ListChildNodeInfoADM, children: Seq[ListChildNodeAD
  * @param labels   the labels of the node in all available languages.
  * @param comments the comments attached to the node in all available languages.
  */
-abstract class ListNodeInfoADM(
-  id: IRI,
-  name: Option[String],
-  labels: StringLiteralSequenceV2,
-  comments: StringLiteralSequenceV2
-) {
+sealed trait ListNodeInfoADM {
```

Comment:
Shouldn't the params from docstring be removed?
Could you remove ADM suffix here too?

Reply:
cleaned up the scala doc: https://github.com/dasch-swiss/dsp-api/pull/3006/commits/532f07569de17d1d8b9870b4e29a4ce6cd6047b6

Renaming was not part of the refactoring.